README.md

```diff
@@ -132,7 +132,11 @@ Alternatively, refer to the [developer instructions](#developer-instructions) fo
                                      caseless sub-string)
     --max-downloads NUMBER           Abort after downloading NUMBER files
     --min-filesize SIZE              Do not download any videos smaller than
-                                     SIZE (e.g. 50k or 44.6m)
+                                     SIZE (e.g. 50k or 44.6m). Use match-filter
+                                     with the duration key instead of this to
+                                     filter for long videos only as this will
+                                     often download the video track and skip
+                                     the audio if they are downloaded separately.
```

Comment:
Can you tell me which file I should edit in order for this to be regenerated correctly? I've just made a change to options.py to reflect the same thing.

2 & 3 are very good points. But I'm sure there are other people who are as silly/lazy as me who try to use the simple option of filtering by file-size instead of duration within match-filter! This added text is here to warn them off that before they waste an hour of their lives trying to figure out why they are only getting the video track! :)

Thanks.